src/pages/home/profile/IdenfyModal.tsx

```diff
@@ -0,0 +1,36 @@
+import { FunctionComponent } from "react";
+import { useSelector } from "react-redux";
+import { selectSession } from "modules/session";
+import { Modal } from "components";
```

Comment:
I noticed that this modal that was brought in from the token sale is full screen. Are we going to keep our modals full screen or did we import the wrong one? @scandycuz

Reply:
I will have a conversation with product bc i feel like the full screen is a better option for iframes as well to allow flexibility vs trying to fit a full page in 600px lol